src/brew.ts

```diff
@@ -67,6 +67,19 @@ const generateAllCasks: Fig.Generator = {
     }));
   },
 };
+const generatorAlias: Fig.Generator = {
```

Comment:
[nit, discussion]
Wouldn't this name be more concise?
```suggestion
const alias: Fig.Generator = {
```

Reply:
@AlanSyue I was following the convention in the file.

May be I could change is to `generateAlias` instead of `generatorAlias`

Reply:
I think `generateAliases` is nice 😀